### PR TITLE
Fix(emailPro): add missing error message on account adding/updating form

### PR DESCRIPTION
**Pull request by rfougier**

### CODE REVIEW

File: src/emailpro/account/add/emailpro-account-add.html [hunk 1]
```diff
@@ -14,26 +14,31 @@
...
                             </select>
                         </div>
-                        <span class="help-block"
-                              data-ng-if="takenEmailError"
-                              data-i18n-static="exchange_tab_ALIAS_taken_error_message"></span>
+                        <small class="help-block"
+                               data-i18n-static="exchange_tab_ALIAS_taken_error_message"
+                               data-ng-if="takenEmailError">
+                        </small>
+                        <small class="help-block"
```
Comment (antleblanc) [inline src/emailpro/account/add/emailpro-account-add.html]: Alternative: [ngMessages](https://docs.angularjs.org/api/ngMessages/directive/ngMessages#examples) :smile:

File: src/emailpro/account/update/emailpro-account-update.html [hunk 2]
```diff
@@ -22,28 +22,32 @@
...
                         </div>
-                        <span class="help-block"
-                              data-ng-if="takenEmailError"
-                              data-i18n-static="exchange_tab_ALIAS_taken_error_message" ></span>
+                        <small class="help-block"
+                               data-i18n-static="exchange_tab_ALIAS_taken_error_message"
+                               data-ng-if="takenEmailError">
+                        </small>
+                        <small class="help-block"
+                               data-ng-repeat="(error, value) in emailProUpdateAccountForm.login.$error"
```
Comment (antleblanc) [inline src/emailpro/account/update/emailpro-account-update.html]: Alternative: [ngMessages](https://docs.angularjs.org/api/ngMessages/directive/ngMessages#examples) :smile:

File: src/emailpro/account/add/emailpro-account-add.html [hunk 3]
```diff
@@ -14,26 +14,31 @@
...
                     <label class="col-md-12 control-label text-left required" for="login"
                            data-i18n-static="exchange_ACTION_add_account_step1_email_label"></label>
                     <div class="col-md-12">
                         <div class="input-group">
                             <input type="text" class="form-control" id="login" name="login"
+                                   data-ng-change="checkTakenEmails()"
                                    data-ng-model="accountToAdd.login"
-                                   data-ng-pattern="/^[-_a-zA-Z0-9]+((\.|\+)[-_a-zA-Z0-9]+)*$/"
-                                   data-ng-change="checkTakenEmails()">
+                                   data-ng-pattern="/^[-_a-zA-Z0-9]+((\.|\+)[-_a-zA-Z0-9]+)*$/">
```
Comment (FredericEspiau) [inline src/emailpro/account/add/emailpro-account-add.html]: Not sure this matches the actual regex from backend, you should ask

File: src/resources/i18n/emailpro/Messages_fr_FR.xml [hunk 4]
```diff
@@ -45,6 +45,8 @@
    <translation id="emailpro_ACTION_update_account_error_message" qtlid="375439">L'adresse e-mail E-mail Pro n'a pas pu être personnalisée. Il est possible qu'une configuration mutualisée (mail, redirection, répondeur) existe avec le même nom.</translation>
    <translation id="emailpro_ACTION_change_password_account_success_message" qtlid="375452">Le mot de passe du compte E-mail Pro va être modifié.</translation>
    <translation id="emailpro_ACTION_update_account_title" qtlid="375387">Modifier le compte E-mail Pro</translation>
+   <translation id="emailpro_ACTION_account_error_pattern">Le compte comporte des caractères invalides</translation>
```
Comment (FredericEspiau) [inline src/resources/i18n/emailpro/Messages_fr_FR.xml]: Which characters are invalid ? Also, I don't think the error texts have ever been validated by Content, we should ask them (@jleveugle)

**Pull request merged**